ext/strings.go

```diff
@@ -280,6 +287,20 @@ func StringsLocale(locale string) StringsOption {
 	}
 }
 
+// Version configures the version of the string library. The version limits which
```

Comment:
Could you update this to `StringsVersion` since other extensions in the `ext` package will likely need to use similar techniques going forward.

Reply:
Done